parts/chapters/subsections/4.3/SKIP300.fodt

```diff
@@ -4205,12 +4204,12 @@ Updated with AFR/TSA Rev-D comments and new keywords.</dc:description><meta:init
      </table:table-row>
     </table:table>
     <text:h text:style-name="Heading_20_4" text:outline-level="4" text:is-list-header="true"><text:bookmark-start text:name="__RefHeading___Toc10367_37011683881711111"/>Description<text:bookmark-end text:name="__RefHeading___Toc10367_37011683881711111"/></text:h>
-    <text:p text:style-name="P59">The SKIP<text:span text:style-name="T20">3</text:span>00 keyword activates skipping of all keywords and input data by the commercial <text:span text:style-name="T20">compositional</text:span> simulator until the ENDSKIP keyword is encountered. All keywords between the SKIP<text:span text:style-name="T20">3</text:span>00 and ENDSKIP keywords are ignored by the commercial <text:span text:style-name="T20">compositional</text:span> simulator. The SKIP<text:span text:style-name="T20">3</text:span>00 keyword is ignored by the commercial <text:span text:style-name="T20">black-oil</text:span> simulator. Each SKIP<text:span text:style-name="T20">3</text:span>00 keyword should be paired with an ENDSKIP keyword. See also the SKIP and SKIP<text:span text:style-name="T20">1</text:span>00 keywords. <text:s/></text:p>
+    <text:p text:style-name="P59">The <text:a xlink:href="#REF_HEADING_KEYWORD_SKIP_11_3">SKIP</text:a><text:span text:style-name="T20">3</text:span>00 keyword activates skipping of all keywords and input data by the commercial <text:span text:style-name="T20">compositional</text:span> simulator until the <text:a xlink:href="#__RefHeading___Toc605472_3199477706">ENDSKIP</text:a> keyword is encountered. All keywords between the <text:a xlink:href="#REF_HEADING_KEYWORD_SKIP_11_3">SKIP</text:a><text:span text:style-name="T20">3</text:span>00 and <text:a xlink:href="#__RefHeading___Toc605472_3199477706">ENDSKIP</text:a> keywords are ignored by the commercial <text:span text:style-name="T20">compositional</text:span> simulator. The <text:a xlink:href="#REF_HEADING_KEYWORD_SKIP_11_3">SKIP</text:a><text:span text:style-name="T20">3</text:span>00 keyword is ignored by the commercial <text:span text:style-name="T20">black-oil</text:span> simulator. Each <text:a xlink:href="#REF_HEADING_KEYWORD_SKIP_11_3">SKIP</text:a><text:span text:style-name="T20">3</text:span>00 keyword should be paired with an <text:a xlink:href="#__RefHeading___Toc605472_3199477706">ENDSKIP</text:a> keyword. See also the <text:a xlink:href="#REF_HEADING_KEYWORD_SKIP_11_3">SKIP</text:a> and <text:a xlink:href="#REF_HEADING_KEYWORD_SKIP_11_3">SKIP</text:a><text:span text:style-name="T20">1</text:span>00 keywords. <text:s/></text:p>
```

Comment:
I think here the initial formatting `SKIP<text:span ..>3</text:span>00` broke the link. The some links  should go to SKIP300 and not SKIP

Reply:
Yes, good catch! I have updated the PR. I removed all span tags I could find related to SKIP100 and SKIP300 in `SKIP300.fodt` and then reapplied the script. Can you check again if I missed some?